src/server/api/auxiaProxyRouter.ts

```diff
@@ -103,31 +162,80 @@ const buildAuxiaProxyResponseData = (auxiaData: AuxiaAPIResponseData): AuxiaProx
     };
 };
 
-export const getAuxiaRouterConfig = async (): Promise<AuxiaRouterConfig> => {
-    const apiKey = await getSsmValue('PROD', 'auxia-api-key');
-    if (apiKey === undefined) {
-        throw new Error('auxia-api-key is undefined');
-    }
+// --------------------------------
+// LogTreatmentInteraction Implementation
+// --------------------------------
 
-    const projectId = await getSsmValue('PROD', 'auxia-projectId');
-    if (projectId === undefined) {
-        throw new Error('auxia-projectId is undefined');
-    }
+const buildLogTreatmentInteractionRequestPayload = (
+    projectId: string,
+    userId: string,
+    treatmentTrackingId: string,
+    treatmentId: string,
+    surface: string,
+    interactionType: string,
+    interactionTimeMicros: number,
+    actionName: string,
+): AuxiaAPILogTreatmentInteractionRequestPayload => {
+    // For the moment we are hard coding the data provided in contextualAttributes and surfaces.
```

Comment:
not sure what this comment means - we seem to be taking the data from the client's payload